### PR TITLE
Make sure we have a string, not a SafeBuffer

### PR DESCRIPTION
**Pull request by mthurman**
See https://github.com/email-spec/email-spec/pull/207 for more information. Essentially, if your HTML message has "&39;" and it's actually a SafeBuffer instead of a string, it can't be converted correctly into "'" by HTMLEntities.

### CODE REVIEW

**Comment by dblock** [comment]
+ tests and CHANGELOG and will take it!

**Comment by mthurman** [comment]
@dblock all set!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
-### 0.1.2 (Next)
+### 0.1.3 (Next)
```
Comment (dblock) [inline CHANGELOG.md]: 0.1.2 hasn't shipped yet, so undo this please, lease 0.1.2 (Next)

File: spec/actionmailer-text/html_to_plain_text_spec.rb [hunk 2]
```diff
@@ -69,6 +69,12 @@
     ')).to eq('cédille garçon & à ñ')
   end
 
+  it 'dencodes html entities from a SafeBuffer' do
```
Comment (dblock) [inline spec/actionmailer-text/html_to_plain_text_spec.rb]: Typo: decodes

File: spec/actionmailer-text/html_to_plain_text_spec.rb [hunk 3]
```diff
@@ -69,6 +69,12 @@
...
+    expect(subject.convert_to_text(ActiveSupport::SafeBuffer.new('
+      c&eacute;dille gar&#231;on &amp; &agrave; &ntilde;
+    '))).to eq('cédille garçon & à ñ')
```
Comment (dblock) [inline spec/actionmailer-text/html_to_plain_text_spec.rb]: merci :)

**Comment by dblock** [comment]
Merged. 

@mthurman Any interest in helping with the gem? Maybe make the next release? Drop me an email with your rubygems username/email if you're interested.

**Pull request merged**